### ISSUE TITLE
Team dysfunctional (not functioning)

### ISSUE THREAD

**Issue by HixOba** [Chingu-cohorts/voyage-wiki#272]
**_About You_**
- Voyage number (5 or 6):  Voyage 6
- Team Name: Geckos-08
- Slack Name: 
- Timezone: UTC+1

**_Issue Description & Expected Outcome:_** 
The Geckos-08 team is not functional, each of two remaining members are doing their own thing (that includes me), and this defeats the ethos and spirit of this wonderful Chingu cohort project. 

**_Symptoms:_**
There has been just one meeting, one member @Kate (Kate Beard) has dropped out since the first week without notice. I  have informed the remaining team member @peeweetje of a need for voice meeting/conference and either get fobbed off, non attendance or outright rejection. I am reluctant to upload my local repo backend updates to remote now as i am yet to get any feedback.

**_Steps to Recreate:_** 
Not sure what i need to say here.

**_Resolution:_** 
Our project, is a sound project, and can still be completed and i would ideally need someone to assist with frontend and backend (Node + Express) and React stacks. I would be just as happy to have only a frontend/React partner aadded who will join me/us and (maybe) @peeweetje to take things forward, a team member who is perhaps better experienced with project and workflow management which i dont have much experience of and one who is willing to have regular (weekly) meetings for better coordination.

Finally, i prefer a situation where a new remote repo is provided for the project, and have left a message for @peeweetje informing her of my intention to make a ticket saying our team is dysfunctional and i may have to leave.

PS
I have continued to add more functionality to the backend in my local repo, and will start doing some React frontend by the weekend so that a basic MVP can happen. But an extra hand is needed.

**Comment by mar-bi** [comment]
Hi!
We had a conversation with @Hixsav and I'd like to help him with a setup of workflow and a front-end part. I hope we still can deliver a basic MVP till the deadline.
May I just join his team repo without leaving my team (bears-01)?

**Comment by peeweetje** [comment]
Hi, it's great that you want to help our team. I am using react and semantic UI on the frontend. I am working on the register and login and would like to finish this part. Let me know what part you would to work on? Maybe you can join our teamchannel so we could have a conversation?

Op 27 jul. 2018 12:35 PM schreef Marina Biletska <notifications@github.com>:

Hi!
We had a conversation with @Hixsav<https://github.com/Hixsav> and I'd like to help him with a setup of workflow and a front-end part. I hope we still can deliver a basic MVP till the deadline.
May I just join his team repo without leaving my team (bears-01)?

**Comment by peeweetje** [comment]
I would like to say I am very dissapointed that @Hixsav doesn't appreciate me as a teammember. I am not unresponsive, and tried to give it my best. Voyage needs to be a positive experience. I am very dissapointed it looks like he doesn't want to work with me anymore.

**Comment by HixOba** [comment]
I am very happy to work with any one who is happy to have regular scheduled
voice or video meetings at least once a week so we can coordinate better. I
am not enthusiastic to keep having to work blind and use brief slack
messages as only line of communication. For me, thats a poor team
experience. I dont have any personal issues otherwise, and thats
being honest.

**Comment by mar-bi** [comment]
Let's enhance your team experience.
I can organize the team workflow and can help with the front-end.
Your goal is, probably, to deliver your app. 
If you both @peeweetje and @Hixsav agree and ready to work together 1 more week, I need access to
your repo. And we will have a meeting. :smiley:

**Comment by HixOba** [comment]
@mar-bi thanks for your offer, a positive experience (or the total lack of it) is why I opted to pull out. If we are able to establish this and setup a streamlined workflow system in the short time left I would definitely be up for it. A key strength of Chingu as i understand it is supposed to be developing a team(work) ethic. I like to discuss issues arising from my branch before it is merged.

**Comment by HixOba** [comment]
HI Marina

Are you on discord? We can chat on there, here is an invite

https://discord.gg/tQ5DQF

I think it expires in 24 hours, if we dont get to chat before it expires
i'll simply generate another one. But let me know when you are able to chat.

Cheers
Obaye

**Comment by HixOba** [comment]
Hi @chance, this has been sorted as agreed. Thanks

**Comment by HixOba** [comment]
HI Marina, how are you getting on? On my local, i have put up a functional
placeholder for landing page and navbar. I have also set react router on
frontend. And am moving towards dashboard, but obviously need some basics
on the dashboard, such as registration form and login form sorted first.

I have added some issues on the workspace because Oliver and Nedu have
signified interest to assist. And they have asked to know what needs doing.
This can be overwritten by whatever you come up with if better, no problem.
Just letting the guys know what needs to be done so they can work on it in
their 'local'.
